io.catenax.time_series_reference/1.0.0/TimeSeriesReference.ttl

```diff
@@ -0,0 +1,156 @@
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#> .
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:bamm:io.catenax.time_series_reference:1.0.0#> .
+
+:TimeSeriesReference a bamm:Aspect ;
+   bamm:preferredName "time series reference"@en ;
+   bamm:properties ( :decimalSeperator :sourceFile :fileExtension :delimiter :payload ) ;
+   bamm:operations ( ) ;
+   bamm:events ( ) ;
+   bamm:name "TimeSeriesReference" .
+
+:decimalSeperator a bamm:Property ;
+   bamm:preferredName "decimal seperator"@en ;
+   bamm:description "indicates whether a dot, comma or semicolon is used to separate the decimal place in the source file"@en ;
+   bamm:characteristic :DecimalSeperatorChoice ;
+   bamm:exampleValue "dot" ;
+   bamm:name "decimalSeperator" .
+
+:sourceFile a bamm:Property ;
+   bamm:preferredName "source file"@en ;
+   bamm:description "contains the URI to the referenced time series file"@en ;
+   bamm:characteristic :ResourcePath ;
+   bamm:name "sourceFile" .
+
+:fileExtension a bamm:Property ;
+   bamm:description "indicates whether csv, none or txt is used as file extension"@en ;
+   bamm:characteristic :FileExtensionChoice ;
+   bamm:exampleValue "csv" ;
+   bamm:name "fileExtension" .
+
+:delimiter a bamm:Property ;
+   bamm:description "indicates whether a semicolon, comma or tab is used as a deliniter between datapoints"@en ;
+   bamm:characteristic :DelimiterChoice ;
+   bamm:exampleValue "semicolon" ;
+   bamm:name "delimiter" .
+
+:payload a bamm:Property ;
+   bamm:preferredName "payload"@en ;
+   bamm:description "contains the information about units and variable names"@en ;
+   bamm:characteristic :PayloadCharacteristic ;
+   bamm:name "payload" .
+
+:DecimalSeperatorChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "dot" "comma" ) ;
+   bamm:name "DecimalSeperatorChoice" .
+
+:ResourcePath a bamm:Characteristic ;
+   bamm:preferredName "Resource Path"@en ;
+   bamm:description "The path of a resource."@en ;
+   bamm:dataType xsd:anyURI ;
+   bamm:name "ResourcePath" .
+
+:FileExtensionChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "csv" "none" "txt" "dat" "data" ) ;
+   bamm:name "FileExtensionChoice" .
+
+:DelimiterChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "semicolon" "comma" "tab" ) ;
+   bamm:name "DelimiterChoice" .
+
+:PayloadCharacteristic a bamm-c:TimeSeries ;
+   bamm:preferredName "payload characteristic"@en ;
+   bamm:dataType :PayloadEntity ;
+   bamm:name "PayloadCharacteristic" .
+
+:PayloadEntity a bamm:Entity ;
+   bamm:preferredName "payload entity"@en ;
+   bamm:description "header payload"@en ;
+   bamm:properties ( :units :variableNames :timeColumnName ) ;
```

Comment:
What does this mean? I assume this entity represents one row in the time series?